java/blade/config.yaml

```diff
@@ -0,0 +1,13 @@
+framework:
+  website: http://lets-blade.com/
```

Comment:
please use `website: lets-blade.com`, **https** wil be automatically added

Reply:
Done.